suites/squid/nvmeof/tier-1_nvmeof_sanity.yaml

```diff
@@ -121,13 +122,16 @@ tests:
           - nqn: nqn.2016-06.io.spdk:cnode1
             serial: 1
             bdevs:
-              count: 1
-              size: 10G
-            listener_port: 5001
+              - count: 1
+                size: 10G
+                ns_create_image: true
+            listeners:
+              - node6
```

Comment:
Could you add node5 too? as part of https://github.com/red-hat-storage/cephci/blob/0a528352b252cd950c2085d61dac4af2922ece11/suites/squid/nvmeof/tier-1_nvmeof_sanity.yaml
Corresponding conf file https://github.com/red-hat-storage/cephci/blob/0a528352b252cd950c2085d61dac4af2922ece11/conf/squid/nvmeof/ceph_nvmeof_sanity.yaml